src/gui/presets.cpp

```diff
@@ -931,6 +986,95 @@ void FurnaceGUI::initSystemPresets() {
       0
     }
   ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Capcom CPS-1", { 
+      DIV_SYSTEM_YM2151, 64, 0, 2,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Jaleco Mega System 1", {
+      DIV_SYSTEM_YM2151, 64, 0, 2,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "NMK Arcade", {
+      DIV_SYSTEM_OPN, 64, 0, 0,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Data East Arcade", {
+      DIV_SYSTEM_OPN, 64, 0, 0,
+      DIV_SYSTEM_OPL2, 64, 0, 0,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Kaneko Arcade", {
```

Comment:
Correctly, "Kaneko Toybox System" (Magical crystals, The Berlin Wall, Explosive Breaker(a.k.a. Bakuretsu Breaker in japan; with OKI bankswitching)) uses this sound system.